src/components/Container.js

```diff
@@ -539,6 +547,56 @@ class Container extends Component {
     acceptTokenSuggestion(chapter, verse, alignmentIndex, token);
   }
 
+  /**
+   * will toggle bookmark on click
+   */
+  handleBookmarkClick() {
+    const {
+      addReminder,
+      tool: {
+        api
+      },
+      tc: {contextId},
+      username,
+    } = this.props;
+    const bookmarked = getCurrentReminders(api.context.store.getState());
```

Comment:
Why are you accessing the state trough the context and directly from redux as props?

Reply:
Good question, should have mapped state to props